### PR TITLE
[Project Submission] Add The Convo Space

### PR DESCRIPTION
**Pull request by anudit**
Stoked to add The Convo Space to the IPFS ecosystem of projects.

### CODE REVIEW

File: projects/The Convo Space.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+|[![alt text](https://theconvo.space/images/poster.png)](https://youtu.be/p5-jQy7vKk8 "The Convo Space")|<img src="https://theconvo.space/images/dashboard.PNG">|
+
+## Target Audience
+Developers and Applications/Websites/MetaVerse
+
+## Rough estimated user base (if applicable)
+Early to say but the potential userbase is in ~Millions
+
+## Github repo
+https://github.com/anudit/convo
```
Comment (autonome) [inline projects/The Convo Space.md:40]: Hi, these repos appear to be closed

**Review by autonome — CHANGES_REQUESTED** [review]

File: projects/The Convo Space.md (hunk 1 above)
Comment (anudit) [reply projects/The Convo Space.md:40]: Ah, there you go @autonome

Comment (autonome) [reply projects/The Convo Space.md:40]: great thanks! Can you apply MIT license? it is required for the grants program.

Comment (anudit) [reply projects/The Convo Space.md:40]: Done @autonome

**Pull request merged**